src/manifest.json

```diff
@@ -1,4 +1,10 @@
 ﻿{
+    "applications": {
+        "gecko": {
+            "id": "addon@darkreader.com",
```

Comment:
I own darkreader.org domain, darkreader.com is under some chemistry manufacturer.